packages/api/src/services/emp-stats.ts

```diff
@@ -1,132 +1,155 @@
 import assert from "assert";
 import * as uma from "@uma/sdk";
+export { BigNumber, utils } from "ethers";
```

Comment:
this file is responsible for calculating tvl/tvm from prices and emp states. Its seen a large refactor to break up the monolithic stats object, which contained both tvl/tvm together, into separate tables for these values. this is because a single timestamp was not sufficient to describe when tvl/tvm are updated.